test/smime-certs/mksmime-certs.sh

```diff
@@ -15,23 +15,23 @@ export OPENSSL_CONF
 
 # Root CA: create certificate directly
 CN="Test S/MIME RSA Root" $OPENSSL req -config ca.cnf -x509 -noenc \
-	-keyout smroot.pem -out smroot.pem -newkey rsa:2048 -days 3650
+	-keyout smroot.pem -out smroot.pem -newkey rsa:2048 -days 36501
```

Comment:
Why extra day?

Reply:
As below - it's CA cert so I made it expiring later than the certificates it certifies.